webapp/components/_new/generic/ResponsiveImage/ResponsiveImage.vue

```diff
@@ -0,0 +1,87 @@
+<template>
+  <img :data-src="lazySrc" :data-srcset="lazySrcset" :style="style" class="responsive-image lozad" />
+</template>
+
+<script>
+import lozad from 'lozad'
+
+export default {
+  name: 'ResponsiveImage',
+  props: {
+    backgroundColor: {
+      type: String,
+      default: '#efefef',
+    },
+    height: {
+      type: Number,
+      default: null,
+    },
+    lazySrc: {
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2020-01-28T22:59:07Z" title="Tuesday, January 28th 2020, 11:59:07 pm +01:00">Jan 28, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -0,0 +1,87 @@
+<template>
+  <img :data-src="lazySrc" :data-srcset="lazySrcset" :style="style" class="responsive-image lozad" />
+</template>
+
+<script>
+import lozad from 'lozad'
+
+export default {
+  name: 'ResponsiveImage',
+  props: {
+    backgroundColor: {
+      type: String,
+      default: '#efefef',
+    },
+    height: {
+      type: Number,
+      default: null,
+    },
+    lazySrc: {
```


So we have to give it a `lazySrc`. I was imagining a `<responsive-image :image="post.image">` component that would take an object (or a url as string) as property and provide the `srcSet` etc. for the lozad image.

Reply:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2020-01-29T09:03:38Z" title="Wednesday, January 29th 2020, 10:03:38 am +01:00">Jan 29, 2020</time>_

---


I think we could give it an object and then we can use a method or computed property to pass into the `data-src`, no?